src/main/java/uk/gov/pay/products/service/PaymentFactory.java

```diff
@@ -1,6 +1,6 @@
 package uk.gov.pay.products.service;
 
-public interface PaymentsFactory {
+public interface PaymentFactory {
 
     PaymentFinder paymentsFinder();
```

Comment:
`paymentsFinder` should be `paymentFinder`

Reply:
Changed. 👍